.github/workflows/release.yml

```
@@ -0,0 +1,315 @@
+          echo "- [Quick Start](https://github.com/query4j/dynamicquerybuilder/blob/v$VERSION/QUICKSTART.md)" >> RELEASE_NOTES.md
+          
+          cat RELEASE_NOTES.md
+
+      - name: Collect release files
+        run: |
+          mkdir -p release-files
+          find ./artifacts -name "*.jar" -type f ! -name "*-plain.jar" -exec cp {} release-files/ \;
+          ls -lh release-files/
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)